### PR TITLE
Docs: Updated instructions on screens for Dashboard Import

### PR DESCRIPTION
**Pull request by peterholmberg**
**What this PR does / why we need it**:
Import dashboard docs needed updating, also updated instructions since they seemed out of date.

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/reference/export_import.md [hunk 1]
```diff
@@ -29,14 +29,14 @@ the dashboard, and will also be added as an required input when the dashboard is
 
 ## Importing a dashboard
 
-To import a dashboard open dashboard search and then hit the import button.
+To import a dashboard click the + icon in the sidemenu, and click "import".
```
Comment (oddlittlebird) [inline docs/sources/reference/export_import.md:32]:
```suggestion
To import a dashboard click the + icon in the side menu, and then click **Import**. 
```

Comment (oddlittlebird) [reply docs/sources/reference/export_import.md:32]: Please acquaint yourself with the [Grafana documentation style guide](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md)

**Review by oddlittlebird — CHANGES_REQUESTED** [review]

File: docs/sources/reference/export_import.md [hunk 2]
```diff
@@ -29,14 +29,14 @@ the dashboard, and will also be added as an required input when the dashboard is
...
 ## Importing a dashboard
 
-To import a dashboard open dashboard search and then hit the import button.
+To import a dashboard click the + icon in the side menu, and then click **Import**. 
 
-{{< docs-imagebox img="/img/docs/v50/import_step1.png" max-width="700px" >}}
+{{< docs-imagebox img="/img/docs/v70/import_step1.png" max-width="700px" >}}
 
 From here you can upload a dashboard json file, paste a [Grafana.com](https://grafana.com) dashboard
 URL or paste dashboard json text directly into the text area.
```
Comment (dprokop) [inline docs/sources/reference/export_import.md:37]:
```suggestion
From here you can upload a dashboard JSON file, paste a [Grafana.com](https://grafana.com) dashboard
URL or paste dashboard JSON text directly into the text area.
```

**Review by dprokop — APPROVED** [review]
👍

**Pull request merged**